routes/forecast.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
+var router  = express.Router();
+const User  = require('../models').User
+const fetch = require('node-fetch');
+var forecastSerializer = require('../serializers/forecastSerializer')
+require('dotenv').config()
+
+router.get('/', (req, res) => {
+  if (req.body.apiKey && req.query.location) {
+    User.findOne({
+      where: {
+        apiKey: req.body.apiKey
+      }
+    })
+    .then( user => {
+      if (user && user.apiKeyActive) {
+        fetch(`https://maps.googleapis.com/maps/api/geocode/json?key=${process.env.GOOGLE_GEOCODE_API_KEY}&address=${req.query.location}`)
+        .then(res => res.text())
+        .then(body => JSON.parse(body)["results"][0]["geometry"]["location"])
```

Comment:
I see you're parsing here, and again in the forecastSerializer.  Is there an advantage there, or could you just leave it stringified all the way through?